### PR TITLE
bindings: ros: Add support for camera connected over USB

### PR DESCRIPTION
**Pull request by andreeasandulescu**
The default connection type is now USB.

Signed-off-by: Andreea Sandulescu <andreea.sandulescu@analog.com>

### CODE REVIEW

File: bindings/ros/aditof_roscpp/src/aditof_utils.cpp [hunk 1]
```diff
@@ -11,13 +11,14 @@ std::shared_ptr<Camera> initCameraEthernet(int argc, char **argv) {
     google::InitGoogleLogging(argv[0]);
     FLAGS_alsologtostderr = 1;
     Status status = Status::OK;
-
+    std::string ip;
     if (argc < 2) {
-        LOG(ERROR) << "No ip provided!";
-        return nullptr;
+        LOG(WARNING) << "No ip provided!";
```
Comment (dNechita) [inline bindings/ros/aditof_roscpp/src/aditof_utils.cpp]: I think INFO is better suited here. Client gets informed that in absence of an IP, an attempt to connect to the camera through USB will be made. Which means the message should be updated to explain these things.

File: bindings/ros/aditof_roscpp/src/aditof_utils.cpp [hunk 2]
```diff
@@ -27,7 +28,14 @@ std::shared_ptr<Camera> initCameraEthernet(int argc, char **argv) {
     }
 
     std::vector<std::shared_ptr<Camera>> cameras;
-    system.getCameraListAtIp(cameras, ip);
+    if (argc < 2) {
```
Comment (dNechita) [inline bindings/ros/aditof_roscpp/src/aditof_utils.cpp]: Why not check if ip is not empty?

File: bindings/ros/aditof_roscpp/src/aditof_utils.cpp (hunk 1 above)
Comment (andreeasandulescu) [reply bindings/ros/aditof_roscpp/src/aditof_utils.cpp]: You're right. Fixed in commit https://github.com/analogdevicesinc/aditof_sdk/pull/250/commits/9f03a7a6556856aa50f986fbbbf5b0ceffd0dab8

File: bindings/ros/aditof_roscpp/src/aditof_utils.cpp (hunk 2 above)
Comment (andreeasandulescu) [reply bindings/ros/aditof_roscpp/src/aditof_utils.cpp]: Fixed in commit https://github.com/analogdevicesinc/aditof_sdk/pull/250/commits/9f03a7a6556856aa50f986fbbbf5b0ceffd0dab8

**Pull request merged**